swift.en.md

```diff
@@ -0,0 +1,340 @@
+# Swift coding style
+
+## Introduction
+
+This document defines the conventions for writing Swift code at Cookpad Inc.
+
+This guide directed for Swift 3.0.2.
+
+## Objective
+
+This guides aim to accomplish these objectives.
+
+1. Clarify purposes 
+1. Improve readability
+1. Keep consistency
+1. Reduce side-effects
+
+## Code style
+
+- **[MUST]** Use four spaces for 1-level of indent.  Do not use the horizontal tab character.
+- **[MUST]** Add linebreak to end of files
+- **[MUST]** Place open brackets on same lines
+
+```swift
+// Bad
+func update() 
+{
+}
+
+// Good
+func update() {
+}
+```
+
+- **[MUST]** Use trailing comma on elements of collections
+    - Because of reducing file diffs
```

Comment:
I still don't understand why people actually care about this. It's so extremely trite.